### PR TITLE
Document proposed call for adoption process

### PR DESCRIPTION
**Pull request by dontcallmedom**
As discussed with chairs, with a few additional clarifications suggested by Elad (in lines 5 and 9)

/cc @eladalon1983

### CODE REVIEW

File: call-for-adoption-process.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+* this process is meant for proposals that are candidate for adoption by the WebRTC Working Group, developed in their own repo (which typically should NOT be a @w3c repo); the said proposals are expected to be marked as CG report or Unofficial Drafts until their formal WG adoption
```
Comment (alvestrand) [inline call-for-adoption-process.md]: Nit: uppercase T. This runs throughout the proposed text.

File: call-for-adoption-process.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+* when the Call for Adoption (CfA) starts, and until its review period ends (typically 7 days) , the repo main branch is frozen
+
+* changes that would emerge from the CfA discussion would be filed as pull requests; these pull requests can be submitted by the reviewer asking for change with prompt reviews by the proposer, or vice versa
```
Comment (alvestrand) [inline call-for-adoption-process.md]: voice: leaving oult "would" makes the text easier to read: "Changes that emerge from the CfA discussion, if any, must be filed as pull requests".

File: call-for-adoption-process.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+* when the CfA period expires, if the Working Group chairs determine that the proposal can be adopted as is, the proposal is adopted, initial editors are named, the main branch is unfrozen and the repo transfered to the w3c github organization, with the document marked as up an Editors Draft
```
Comment (alvestrand) [inline call-for-adoption-process.md]: Nit: marked as up -> marked up as

File: call-for-adoption-process.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+
+* if instead there are changes that would be required for the adoption to proceed, and assuming the proposer agrees with the said changes, the relevant pull requests get merged, and another CfA period is started to confirm the resulting document satisfies all parties, with an expectation that the reviews made during the new period would focus on the said changes.
```
Comment (alvestrand) [inline call-for-adoption-process.md]: Nit: confirm the resulting -> confirm the resulting
Style: "with an expectation" -> "The reviews made during the new period should focus on the changes made subsequent to the first CfA."

**Review by alvestrand — APPROVED** [review]
Sounds good to me in substance, quite a few requested tweaks to language.

File: call-for-adoption-process.md (hunk 1 above)
Comment (dontcallmedom) [inline call-for-adoption-process.md]:
```suggestion
* This process is meant for proposals that are candidate for adoption by the WebRTC Working Group, developed in their own repo (which typically should NOT be a @w3c repo); the said proposals are expected to be marked as CG report or Unofficial Drafts until their formal WG adoption.
```

File: call-for-adoption-process.md [hunk 5]
```diff
@@ -0,0 +1,11 @@
+* this process is meant for proposals that are candidate for adoption by the WebRTC Working Group, developed in their own repo (which typically should NOT be a @w3c repo); the said proposals are expected to be marked as CG report or Unofficial Drafts until their formal WG adoption
+
+* when the Call for Adoption (CfA) starts, and until its review period ends (typically 7 days) , the repo main branch is frozen
```
Comment (dontcallmedom) [inline call-for-adoption-process.md]:
```suggestion
* When the Call for Adoption (CfA) starts, and until its review period ends (typically 7 days) , the repo main branch is frozen.
```

File: call-for-adoption-process.md (hunk 2 above)
Comment (dontcallmedom) [inline call-for-adoption-process.md]:
```suggestion
* Changes that emerge from the CfA, if any, discussion must be filed as pull requests; these pull requests can be submitted by the reviewer asking for change with prompt reviews by the proposer, or vice versa.
```

File: call-for-adoption-process.md (hunk 3 above)
Comment (dontcallmedom) [inline call-for-adoption-process.md]:
```suggestion
* When the CfA period expires, if the Working Group chairs determine that the proposal can be adopted as is, the proposal is adopted, initial editors are named, the main branch is unfrozen and the repo transfered to the w3c github organization, with the document marked up as an Editors Draft.
```

File: call-for-adoption-process.md [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+
+* changes that would emerge from the CfA discussion would be filed as pull requests; these pull requests can be submitted by the reviewer asking for change with prompt reviews by the proposer, or vice versa
+
+* when the CfA period expires, if the Working Group chairs determine that the proposal can be adopted as is, the proposal is adopted, initial editors are named, the main branch is unfrozen and the repo transfered to the w3c github organization, with the document marked as up an Editors Draft
+
+* if instead there are changes that would be required for the adoption to proceed, and assuming the proposer agrees with the said changes, the relevant pull requests get merged, and another CfA period is started to confirm the resulting document satisfies all parties, with an expectation that the reviews made during the new period would focus on the said changes.
+
+* whenever a CfA period expires without an agreement to adopt the proposal, the proposer or the chairs may decide to stop or pause the adoption review process, at which point the repo main branch gets unfrozen
```
Comment (dontcallmedom) [inline call-for-adoption-process.md]:
```suggestion
* Whenever a CfA period expires without an agreement to adopt the proposal, the proposer or the chairs may decide to stop or pause the adoption review process, at which point the repo main branch gets unfrozen.
```

File: call-for-adoption-process.md (hunk 4 above)
Comment (dontcallmedom) [inline call-for-adoption-process.md]:
```suggestion
* If instead there are changes that would be required for the adoption to proceed, and assuming the proposer agrees with the said changes, the relevant pull requests get merged, and another CfA period is started to confirm the resulting document satisfies all parties. The reviews made during the new period would focus on the said changes.
```

**Pull request merged**